.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -4,7 +4,19 @@ about: Report a bug encountered with the gateway-api project
 labels: kind/bug
 
 ---
-<!-- Please use this template while reporting a bug and provide as much info as
+<!--
```

Comment:
Since we're about so say "go away!" maybe we could lead with "Thank you for your interest in Gateway API!" ?

Reply:
Good call, updated, thanks!